gtrending/fetch.py

```diff
@@ -54,14 +54,14 @@ def fetch_repos(
 
 
 def fetch_developers(language: str = "", since: str = "daily") -> dict:
-    """Fetch trending developers on GitHub
+    """Fetch the trending developers on GitHub.
```

Comment:
I think it's a bit extra
```suggestion
    """Fetch trending developers on GitHub.
```